x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/action/RestDeleteConnectorAction.java

```diff
@@ -35,7 +35,11 @@ public List<Route> routes() {
 
     @Override
     protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient client) throws IOException {
-        DeleteConnectorAction.Request request = new DeleteConnectorAction.Request(restRequest.param("connector_id"));
+
+        String connectorId = restRequest.param("connector_id");
+        boolean shouldDeleteSyncJobs = restRequest.paramAsBoolean("delete_sync_jobs", true);
```

Comment:
Maybe double-confirm the default value with product keeping for example auditing purposes in mind, where you never want to loose sync jobs

Reply:
And also for backward-compatibility

Reply:
@danajuratoni  suggested to follow default Kibana behaviour. 
Defaulting this to false is the default in Kibana, also it's backward compatible, so I will follow that approach.